sof_wrapper/api/fhir.py

```diff
@@ -1,10 +1,47 @@
-from flask import Blueprint, current_app
+from flask import Blueprint, current_app, session
 import requests
 
 
 blueprint = Blueprint('fhir', __name__, url_prefix='/v/r2/fhir/')
```

Comment:
@ivan-c I think we should do away with this url_prefix, as the contained APIs support multiple versions.  Either inline the prefix or break this into views by version?

Reply:
@pbugni 
>Either inline the prefix

I think this makes the most sense for flexibility. We don't expose every attribute of the FHIR objects we support and some may not change between FHIR versions